include/k4ainternal/k4aplugin.h

```diff
@@ -23,13 +23,28 @@ extern "C" {
 #endif
 
 /**
- * Current Version of the k4A Depth Engine Plugin API
+ * Current Version of the K4A depth engine and this Plugin interface
  *
- * \remarks The MAJOR version must be updated to denote any breaking change
- * that would cause an older SDK to not be able to use this plugin.
+ * \remarks
+ * This plugin version and the depth engine are tied together with the major.minor version. When the depth engine takes
```

Comment:
These remarks mark a difference in the way I was thinking about the depth engine. I think I see the way you are looking at it, but I just want to make sure we are on the same page.

My view
The K4A SDK defines a plugin interface. Anyone can build an implementation against that interface. Since the SDK defines the interface, the SDK also defines the interface version. Any implementation of the interface should take its major and minor version from the interface version defined by the SDK.

What I believe I reading here
The depth engine is the golden standard for the plugin interface. We build an interface to allow others to implement there own plugin, but the root of truth is our depth engine. If our depth engine version changes, so does our plugin interface.

I'm fine either way, just want to make sure I'm understanding correctly.